frontend/app/onboard.tsx

```diff
@@ -0,0 +1,245 @@
+            placeholder="Enter Password"
+            secureTextEntry={!passwordVisible}
+            value={password}
+            onChangeText={setPassword}
+          />
+          <TouchableOpacity
+            style={styles.eyeIcon}
+            onPress={() => setPasswordVisible(!passwordVisible)}
+          >
+            <Ionicons
+              name={passwordVisible ? "eye-off" : "eye"}
+              size={24}
+              color="gray"
+            />
+          </TouchableOpacity>
+          <TextInput
+            style={styles.input}
+            placeholder="Confirm Password"
+            secureTextEntry={!passwordVisible}
+            value={confirmPassword}
+            onChangeText={setConfirmPassword}
+          />
+          {!passwordMatch && (
+            <Text style={styles.errorText}>Passwords do not match.</Text>
+          )}
+        </View>
+      ) : currentSlide === 4 ? (
+        <View style={styles.buttonGroup}>
+          <CustomButton
+            text={"Log in with Spotify"}
+            onPress={() => alert("Open with Spotify")}
+            backgroundColor="#1DB954"
+          />
+          <CustomButton
+            text={"Log in with Apple"}
+            onPress={() => alert("Open with Apple Music")}
+          />
+        </View>
+      ) : (
+        <View>
+          {tried && !inputValid && (
+            <Text style={styles.errorText}>Is this correct?</Text>
+          )}
+          <TextInput
+            style={[
+              styles.input,
+              !inputValid && tried && { borderColor: "#8b0000" },
+            ]}
+            keyboardType={currentSlide === 1 ? "email-address" : "default"}
+            placeholder={slides[currentSlide].placeholder}
+            autoComplete={currentSlide === 1 ? "email" : "off"}
+            value={
+              currentSlide === 0
+                ? name
+                : currentSlide === 1
+                  ? email
+                  : currentSlide === 3
+                    ? username
+                    : ""
+            }
+            onChangeText={
+              currentSlide === 0
+                ? setName
+                : currentSlide === 1
+                  ? setEmail
+                  : setUsername
+            }
+            onBlur={() => {
+              if (currentSlide === 1) {
+                setInputValid(emailFormat.test(email));
+              } else if (currentSlide === 0 || currentSlide === 3) {
+                setInputValid(true);
+              }
+              setTried(false);
+            }}
+          />
+        </View>
+      )}
+
+      <View style={styles.stickyContainer}>
+        <CustomButton text="Continue" onPress={handleNext} />
+        <ProgressBar
+          progress1={progressBar1}
+          progress2={progressBar2}
+          currentSlide={currentSlide}
+          handleSlideChange={handleNext}
+        />
+      </View>
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  container: {
```

Comment:
Make the container have a white background. RIght now It is black and you can't see what you are typing.